ajax/libs/aquarelle/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "aquarelle",
+  "filename": "aquarelle.min.js",
+  "homepage": "https://ramotion.com/gthb",
+  "description": "watercolor animation",
+  "keywords": [
+    "aquarelle",
+    "watercolor",
+    "animation",
+    "transition"
```

Comment:
![2017-12-17 12 14 33](https://user-images.githubusercontent.com/8349587/34076487-e06b1fae-e323-11e7-9d1c-0978ed71c572.png)
Maybe we can add more keywords.
Thanks